### PR TITLE
Added more bullshit to the bullshit.

### PR DESCRIPTION
**Pull request by regexident**

### CODE REVIEW

File: src/terms.js [hunk 1]
```diff
@@ -3,12 +3,20 @@ var bullshitTerms = [
...
     '24/7',
+    '(benefit|gap) analysis',
+    '(business.(to|2).business)'
+    '(high|mass).impact',
+    '(mission|time).critical',
+    '(search engine )?optimization',
+    '(total cost of )?ownership',
     'a-b testing',
     'accessib(le|ility)',
+    'accessibility'
```
Comment (mourner) [inline src/terms.js]: covered by the regexp ^^^

File: src/terms.js [hunk 2]
```diff
@@ -68,17 +80,20 @@ var bullshitTerms = [
...
     'downsize',
     'drill down',
     'drink the kool-aid',
-    'drop.?in',
     'drop the ball',
+    'drop.?in',
     'due dilligence',
     'dynamic',
     'e-?(business|commerce|tailers)',
+    'e-?commerce'
```
Comment (mourner) [inline src/terms.js]: also covered one line above

File: src/terms.js [hunk 3]
```diff
@@ -285,33 +302,39 @@ var bullshitTerms = [
     'team.player',
     'teamwork',
     'technologies',
+    'testmanagement'
```
Comment (mourner) [inline src/terms.js]: no space - intentional?

File: src/terms.js [hunk 4]
```diff
@@ -68,17 +80,20 @@ var bullshitTerms = [
     'delight the customers?',
     'deliverables?',
     'demographic',
+    'digital(es)? marketing'
+    'digitale? transformaion'
```
Comment (mourner) [inline src/terms.js]: typo in transformation

**Comment by regexident** [comment]
All fair points. Unfortunately I deleted the fork prematurely and thus cannot fix those. :/
Would you fix them, or should I make another PR from another fork? Sorry for inconvenience.

**Comment by regexident** [comment]
All fair points. Unfortunately I deleted the fork prematurely and thus cannot fix those. :/
Would you fix them, or should I make another PR from another fork? Sorry for trouble.

**Comment by mourner** [comment]
Yeah, I can fix — thanks!

**Comment by regexident** [comment]
Thanks!

Turns out my first name "Vincent" gets tagged as "V**bullshit**".
That was — ehm — unexpected. :hushed:

**Comment by mourner** [comment]
@regexident ha, that looks like a bug :)

**Pull request closed without merging**